source/_posts/2021-02-03-release-20212.markdown

```diff
@@ -45,52 +54,174 @@ Issues introduced in the beta are processed with priority.
 - [All changes](#all-changes)
 
+## Erik Montnemery joins Nabu Casa
+
+Today, we are proud to announce that Erik Montnemery has joined Nabu Casa
+to work full-time on the Home Assistant project.
+
+Some of you might know Erik better by his GitHub handle: [@emontnemery], or
+by one of the integrations he works on: [Cast](/integrations/cast),
+[Tasmota](/integrations/tasmota) and [MQTT](/integrations/mqtt)!
+
+We are excited to have you aboard Erik! Welcome!
+
 ## Python 3.7 dropped, Python 3.9 supported
 
 Support for Python 3.7 has been removed, raising the minimum Python version
 you need to run Home Assistant Core to Python 3.8. Python 3.7 was already
 deprecated since Home Assistant Core 0.116.0.
 
 This is only a concern if you are running Home Assistant Core directly on
-Python. if you run Home Assistant OS, Supervised or Container, you don’t have
+Python. If you run Home Assistant OS, Supervised or Container, you don’t have
 to worry about this, as the Home Assistant project takes care of this for you.
 
 However, say hello to: Python 3.9. As of this release, Python 3.9 is fully
 supported and tested.
 
 ## Z-Wave JS
 
-<!-- TODO 
-- Extend paragraph
-- mention working
-- mention add-on
-- explain where it came from
-- thank a lot of people for this amazing achievement
-
-Contributors:
-- AlCalzone
-- https://github.com/robertsLando
-- https://github.com/Michsior14
-- https://github.com/kpine
-- https://github.com/jcam
-- https://github.com/itewk
-- https://github.com/Petro31
-- https://github.com/bramkragten
+This release brings you the [Z-Wave JS integration](/integrations/zwave_js)!
+
+A brand new integration for Z-Wave in Home Assistant, full-blown, created
+in a just month by a bunch of awesome people!
+
+More and more people were concerned about the future of Z-Wave with
+Home Assistant; meanwhile the [Z-Wave JS][zwavejs] project was rapidly growing
+and gathering a large community around it. Long story short: Home Assistant and
+Z-Wave JS teamed up! And a lot of contributors jumped on the train!
+
+This new integration is based on the same base principles as the OpenZWave
+integration: It is decoupled from Home Assistant. Instead of MQTT, the Z-Wave
+JS integration uses a WebSocket connection to a Z-Wave JS server.
+
+This means, in order to use this new integration, you'll need to run the
+Z-Wave JS server that sits in between your Z-Wave USB stick and Home Assistant.
+There are multiple options available on running the Z-Wave JS server,
+via Docker, manually, but there is also an Home Assistant add-on available.
+
+If you run an installation with a Supervisor, the add-on will even be
+set up automatically for you.
+
+[zwavejs]: https://zwave-js.github.io/node-zwave-js/#/
+
+### Current state of Z-Wave JS
+
+The new Z-Wave JS integration is amazingly complete. It supports the following
+platforms from the start: Binary Sensors, Climate, Covers, Fans, Lights, Locks,
+Sensors and Switches.
```

Comment:
Scene events are also supported and maybe you want to mention it is REALLY FAST ;-)

Reply:
Done!